From: John Wiegley <johnw@newartisans.com>
Date: Mon, 21 Apr 2014 18:01:08 -0500
Subject: Initialize journal_t::no_aliases to false upon costruction

This fixes a bug introduced by ecd5097d515f53703eb5dc6096da80182c452ad9.
---
 src/journal.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/journal.cc b/src/journal.cc
index 804c9ee..142b68a 100644
--- a/src/journal.cc
+++ b/src/journal.cc
@@ -97,6 +97,7 @@ void journal_t::initialize()
   day_break         = false;
   checking_style    = CHECK_NORMAL;
   recursive_aliases = false;
+  no_aliases        = false;
 }
 
 void journal_t::add_account(account_t * acct)
